File: app/src/main/java/com/cyberIyke/allvideodowloader/database/ShortcutTable.java
package com.cyberIyke.allvideodowloader.database;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "ShortcutTable")
public class ShortcutTable {
    @PrimaryKey(autoGenerate = true)
    public int id;

    public Integer imgLogo;
    public String strTitle;
    public String strURL;

    public ShortcutTable(Integer imgLogo, String strTitle, String strURL) {
        this.imgLogo = imgLogo;
        this.strTitle = strTitle;
        this.strURL = strURL;
    }
}

File: app/src/main/java/com/cyberIyke/allvideodowloader/database/ShortcutTable.kt
package com.cyberIyke.allvideodowloader.database

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "ShortcutTable")
class ShortcutTable(var imgLogo: Int, var strTitle: String, var strURL: String) {
    @PrimaryKey(autoGenerate = true)
    var id: Int = 0
}